### PR TITLE
[EDU-2448] refactor: PT - cache keys into edge caching

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Removed cache key format information from RTP
- Added cache key section to edge caching page

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-2448]: https://aziontech.atlassian.net/browse/EDU-2448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx [hunk 1]
```diff
@@ -27,6 +27,99 @@ Na Azion, você pode personalizar o tempo que o cache permanece no edge através
...
+```
+
+Uma resposta bem-sucedida retornará um cabeçalho `X-Cache-Key` com o valor da cache key, que com base no exemplo acima, seria:
+
+```
+X-Cache-Key: httpsseudominio.com/caminho-do-recurso/imagem.jpeg@@
+```
+
+:::note
+Saiba mais sobre como obter outros dados de cache no guia [Como verificar indicadores de cache usando Modheader para Google Chrome](/documentacao/produtos/guias/verificar-tempo-de-cache-da-pagina/).
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx:47]:
```suggestion
Saiba mais sobre como obter outros dados de cache no guia [Como verificar indicadores de cache usando Modheader para Google Chrome](/pt-br/documentacao/produtos/guias/verificar-tempo-de-cache-da-pagina/).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx [hunk 2]
```diff
@@ -27,6 +27,99 @@ Na Azion, você pode personalizar o tempo que o cache permanece no edge através
...
+
+O formato padrão da cache key adotado pela Azion concatena os seguintes elementos de sintaxe da URI:
+
+- Schema da requisição
+- Host
+- Path do recurso
+- Query strings
+- Um separador de variação e, quando implementadas, variações
+
+Um objeto com o URI `https://static.seudominio.com/pagina/site.js` deveria gerar a cache key `httpsstatic.seudominio.com/pagina/site.js`.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx:60]:
```suggestion
Um objeto com a URI `https://static.seudominio.com/pagina/site.js` deveria gerar a cache key `httpsstatic.seudominio.com/pagina/site.js`.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx [hunk 3]
```diff
@@ -27,6 +27,99 @@ Na Azion, você pode personalizar o tempo que o cache permanece no edge através
...
+
+#### Requisições complexas
+
+Se o recurso é obtido usando uma requisição complexa (qualquer método diferente de `GET` ou `HEAD`), o método é adicionado no início da cache key. 
+
+Por exemplo, para uma requisição `OPTIONS`, a cache key gerada para um objeto seria `optionshttpsstatic.seudominio.com/pagina`.
+
+#### Image Processor
+
+Quando o [Image Processor](/pt-br/documentacao/produtos/edge-application/image-processor/) está ativado, além da query string `ims`, variações com formato convertido contém o formato do arquivo após o separador.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/edge-caching.mdx:72]:
```suggestion
Quando o [Image Processor](/pt-br/documentacao/produtos/edge-application/image-processor/) está ativado, além da query string `ims`, variações com formato convertido contêm o formato do arquivo após o separador.
```

**Review by hannah-sk — COMMENTED** [review]
deixei algumas correções de PT

**Review by hannah-sk — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**